README.md

```diff
@@ -1,6 +1,6 @@
 # Google Student Training in Engineering Program
 
-This repo contains the projects you'll work on for the first half of STEP.
+This repo Hannah Salameh's portfolio and STEP projects.
```

Comment:
I think you removed the word "contains" by mistake

Reply:
I'll go in and fix that. Also I'm not sure how to respond to your other comments, but I am pretty sure this is in a new repo. It says its unlinked from the google repo at least.

Reply:
okay that sounds good then! Thanks!